### PR TITLE
Write out splits + fix for videos that cannot be loaded

### PR DESCRIPTION
**Pull request by ejm714**
If you're finetuning and having splits autogenerated, you're going to want to know which files were used for training vs. holdout, particularly so you can run your trained model on the holdout/test set. 

Changes:
- writes out filepath and split column to csv in save_dir / splits.csv
- incorporates fix for videos that cannot be loaded (warn and write all zeros)

Before merge:
- [ ] add test for splits files getting written out
- [ ] add test for bad video getting zeroes

### CODE REVIEW

**Review by pjbull — APPROVED** [review]

**Comment by pjbull** [comment]
OPened #122 for outstanding tests here
